### ISSUE TITLE
Ordered list with alphabet doesn't work 

### ISSUE THREAD

**Issue by robyhuzwandar** [danomatic/react-pdf-html#44]
I have an HTML tag like this, which I generated using TinyMCE editor

<ol style="list-style-type: lower-alpha;">
<li style="color: rgb(0, 0, 0); font-family: OpenSans;"> <span style="font-family: OpenSans;">this a</span>
     </li>
<li style="color: rgb(0, 0, 0); font-family: OpenSans;"> <span style="font-family: OpenSans;">this b</span>
</li>
</ol>

and this is the result on the pdf

<img width="218" alt="image" src="https://user-images.githubusercontent.com/20414047/210166277-a5769231-0395-471c-b3c9-ebbb6ce4d845.png">

this my code is very basic:

```js
  <View style={{ marginLeft: "28px" }}>
          <Html
            style={{
              fontSize: 10,
            }}
          >
            {getHtml(text)}
          </Html>
  </View>

```

thisis my html tags I try using w3school

<img width="935" alt="image" src="https://user-images.githubusercontent.com/20414047/210166314-48239108-b061-4c60-90b0-049bd4a7a1ba.png">

get this warning info on console
<img width="635" alt="image" src="https://user-images.githubusercontent.com/20414047/210171191-d916dcdd-b0b7-4003-86c7-f19c13f33176.png">

I use library version:
  "react-pdf-html": "^1.1.15",
   "@react-pdf/renderer": "^3.0.1",
   
   
 can anyone help me ? 
cc :  @danomatic @pmelab @Leksat

**Comment by robyhuzwandar** [comment]
Halo : @danomatic can you review my code to solve this issue

https://github.com/danomatic/react-pdf-html/pull/46

**Comment by danomatic** [comment]
Thanks @robyhuzwandar! It's merged. I'll tag and release it shortly.

**Comment by robyhuzwandar** [comment]
Okay, thanks @danomatic, waiting for the new release.

**Comment by robyhuzwandar** [comment]
I hope you have time to release it soon @danomatic

**Comment by danomatic** [comment]
@robyhuzwandar this has been published as `v1.1.16`
